src/main/resources/static/app/components/admin/labs.html

```diff
@@ -43,10 +43,6 @@ <h4 class="list-group-item-heading" ng-click="toggleVisibility(lab)">
                 <button class="btn btn-default" type="button" title="Edit" ng-click="edit(lab)">
                     Edit
                 </button>
-                <button class="btn btn-default" type="button" title="Remove"
-                    ng-click="remove(lab)">
-                <span class="glyphicon glyphicon-trash"></span>
-                </button>
             </span>
```

Comment:
I thought removal will still exist although the process is not exactly removing?